google-map.js

```
@@ -959,15 +961,26 @@ Polymer({
     if(this.map && this.enableMarkersClustering) {		
       let options = {
         map: this.map,
-        markers: this.markers
+        markers: this._getClusterableMarkers()
       };
       if (typeof this.customRenderer === 'string') {
         options.renderer = eval(this.customRenderer);
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)